ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -209,6 +219,29 @@ Wait For Pods Numbers
         Run Keyword And Continue On Failure    FAIL    Timeout- ${output} pods found with the label selector ${label_selector} in ${namespace} namespace
   END
 
+Apply DSCInitialization CustomResource
+    [Documentation]
+    [Arguments]        ${dsci_name}=${DSCI_NAME}
+    ${file_path} =    Set Variable    tasks/Resources/Files/
+    Log to Console    Requested Configuration:
+    Create DSCInitialization CustomResource Using Test Variables
+    ${yml} =    Get File    ${file_path}dsci_apply.yml
+    Log To Console    Applying DSCI yaml
+    Log To Console    ${yml}
+    ${return_code}    ${output} =    Run And Return Rc And Output    oc apply -f ${file_path}dsci_apply.yml
+    Log To Console    ${output}
+    Should Be Equal As Integers  ${return_code}  0  msg=Error detected while applying DSCI CR
+    Remove File    ${file_path}dsci_apply.yml
+
+Create DSCInitialization CustomResource Using Test Variables
+    [Documentation]
+    [Arguments]    ${dsci_name}=${DSCI_NAME}
+    ${file_path} =    Set Variable    tasks/Resources/Files/
+    Copy File    source=${file_path}dsci_template.yml    destination=${file_path}dsci_apply.yml
+    Run    sed -i 's/<dsci_name>/${dsci_name}/' ${file_path}dsci_apply.yml
+    Run    sed -i 's/<application_namespace>/${APPLICATIONS_NAMESPACE}/' ${file_path}dsci_apply.yml
+    Run    sed -i 's/<monitoring_namespace>/${MONITORING_NAMESPACE}/' ${file_path}dsci_apply.yml
```

Comment:
instead of using sed we have cleaner way: `Create File From Template`. It applies the value of the variables in a given yaml

```yaml
KEY_1: ${value_1}
KEY2: ${value_2}
...
```
```
Set Task Variable    ${value_1}     myval1
Set Task Variable    ${value_2}     myval2
Create File From Template    <template-yaml-path>    <output-yaml-file-path>
```

You can find example by searching for `Create File From Template` in ods-ci.

Anyways, feel free to leave sed commands, work as well